packages/xstate-wallet/src/messaging.ts

```diff
@@ -239,18 +239,33 @@ export function sendDisplayMessage(displayMessage: 'Show' | 'Hide') {
   window.parent.postMessage(message, '*');
 }
 
+export function convertToInternalParticipant(participant: {
+  destination: string;
+  signingAddress: string;
+  participantId: string;
+}): Participant {
+  return {
+    ...participant,
+    destination: makeDestination(participant.destination)
```

Comment:
Nit, maybe keep the naming scheme consistent between `convertToInternalParticipant` and `makeDestination`? In other words, maybe `convertToInternalParticipant` should be `makeParicipant`?

Reply:
I think I might stick with `convertToInternalParticipant`, since it matches `convertToInternalEvent`?